pattern-library/html/boilerplate-header+footer.html

```diff
@@ -0,0 +1,89 @@
+<!--
+edX Pattern Library: Boilerplate HTML Template: Header + Footer
+
+NOTES:
+* base template with the additon of how to handle an app-centric header and footer semantically.
+* this assumes that the header and footer need to have their own visual band/are full width against the viewport. If that's not needed, the ``wrapper-*`` elements around them can be removed in favor of the ``wrapper-view`` grid/layout/visual control.
+-->
+
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+    <head>
+        <meta charset="utf-8">
+        <!-- always force latest IE rendering engine or request chrome rrame -->
+        <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+
+        <base href="/">
+
+        <!-- place favicon.ico in the root directory -->
+        <link rel="shortcut icon" href="favicon.ico">
+        <!-- place touch icon in the root directory -->
+        <link rel="apple-touch-icon" href="apple-touch-icon.png">
+
+        <!-- view/page metadata -->
+        <title></title>
```

Comment:
I like the convention of using {stuff-should-go-here}, so how about making this:

    <title>{page-title}</title>